### DISCUSSION TITLE
Vue组件引用失败

### DISCUSSION THREAD

**Discussion by FoundTheWOUT** [the1812/Bilibili-Evolved#1678] [提问 / Q&A]
## 情况
父组件渲染成功, 且成功插入DOM, 子组件import没有错误, 但引用时出现以下错误
![截屏2021-03-26 上午10 46 14](https://user-images.githubusercontent.com/47922588/112570889-d8f60980-8e21-11eb-8207-d77da8d53be2.png)
## 目录结构
![截屏2021-03-26 上午10 45 13](https://user-images.githubusercontent.com/47922588/112570809-b7951d80-8e21-11eb-86d7-38fc5c57eed7.png)
## 父组件
![截屏2021-03-26 上午10 44 38](https://user-images.githubusercontent.com/47922588/112570849-c7146680-8e21-11eb-97bf-03dfb1ea918a.png)
## 子组件
![截屏2021-03-26 上午10 45 35](https://user-images.githubusercontent.com/47922588/112570877-d09dce80-8e21-11eb-93ed-752e3240b452.png)

**Comment by the1812** [comment]
删掉那个空的`<style>`看看?

**Reply by FoundTheWOUT** [reply]
不太行, 错误还是一摸一样, 还试过填充style, 还是出现一样的错误. 重编译也试过了. 问题依然...

**Reply by FoundTheWOUT** [reply]
这是父组件的注入方式
![截屏2021-03-26 下午12 18 41](https://user-images.githubusercontent.com/47922588/112576548-7571d900-8e2d-11eb-8cd6-85d01b9ac301.png)

**Reply by the1812** [reply]
你把代码传你自己分支上吧, 我之后帮你调试下

**Comment by LonelySteve** [comment]
我觉得这个评论区无限列表的功能，官方以后可能会加上吧，脚本来实现的话，恐怕很麻烦，你还要考虑到与简化评论区功能的兼容，有什么好的方法吗？

**Reply by FoundTheWOUT** [reply]
确实是这样，简化评论区和黑暗模式的适配我还没考虑的，我的思路其实是尽量还原官方样式的（除了回复区，打算是设计一个sticky），之后简化评论区和黑暗评论区可以尝试插入一下

**Reply by the1812** [reply]
重新写了个类似官方的评论区吗? 很多功能都没法直接调用吧
简化和夜间, 只要 class 和原版一样就能直接适配
~~我也是想等官方实现的~~

**Reply by FoundTheWOUT** [reply]
我就说怎么没人搞...原来都在等官方实现🥲, 功能是指..进入别人空间, 回复之类的吗, 感觉确实有点麻烦, 如果只是获取的话, 还是比较简单的

**Reply by the1812** [reply]
很多... 头像点进空间 停留出现个人资料卡 还有举报 删评 拉黑 复制链接
这工作量非常大的

**Reply by LonelySteve** [reply]
是啊，就从可行性而言，应该是没问题的，就是要评估一下做这功能的投入，基本上就是需要重写一遍评论区，然而这功能真的有一定可能被官方做出来，白费那么大功夫（

**Reply by FoundTheWOUT** [reply]
那还是先观察一段时间吧(逃 ... (或许会有更简单的实现方式, 比如拿住之前评论, 然后重新插进去

**Comment by FoundTheWOUT** [comment]
惊了, b站居然实装了无限评论区, 然后回复区还真做成了sticky....

**Reply by LonelySteve** [reply]
说不定有内鬼（

**Reply by LonelySteve** [reply]
好了，这个功能官方替我们实现了，但是现在又有人想改回原来分页的那种（#1748）。。。无论是在原来分页的基础上加上无限滚动加载，还是在现在无限滚动加载的基础上改回分页，都挺麻烦的😂

**Reply by the1812** [reply]
(摸了
